### DISCUSSION TITLE
Outputting Total Gibbs Free Energy in Phase Field Simulations

### DISCUSSION THREAD

**Discussion by ppandit95** [idaholab/moose#20241] [Q&A General] [answered]
Dear MOOSE Comunity

I am working with the martensite transformation test input deck which I have modified as per the requirements but I wish to get the values of gibbs energy at each time step so that II can verify my simulation from the results in the literature.Any headers in this regard will prove beneficial.

Many Thanks 
Pushkar

**Comment by ykvishal** [comment, marked as answer]
Check this [TotalFreeEnergy](https://mooseframework.inl.gov/source/auxkernels/TotalFreeEnergy.html). More options are available on [AuxKernels System](https://mooseframework.inl.gov/syntax/AuxKernels/index.html) page.

**Reply by ppandit95** [reply]
Thanks @ykvishal for directing to the relevent webpage
